File: user-service/src/main/java/com/codePro/service/VO/Department.scala
package com.codePro.service.VO

import lombok.{AllArgsConstructor, Data, NoArgsConstructor}

import scala.beans.BeanProperty

@Data
@AllArgsConstructor
@NoArgsConstructor
class Department {
  @BeanProperty var departmentId: Long =_
  @BeanProperty var departmentName: String =_
  @BeanProperty var departmentAddress: String =_
  @BeanProperty var departmentCode: Long =_
}

File: department-service/src/main/java/com/codePro/department/entity/Department.java
package com.codePro.department.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Department {
    @Id
    @Column(name = "id", nullable = false)
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long departmentId;
    private String departmentName;
    private String departmentAddress;
    private String departmentCode;

}
